### ISSUE TITLE
2.3.1

### ISSUE THREAD

**Issue by shimzim** [jimschubert/swagger-codegen-cli.docker#5]
Hi,

I saw you have a 2.2.3 tag released to dockerhub, and latest. Are you planning to release a 2.3.1 or 2.3.0 tag to dockerhub?

Thanks!

**Comment by jimschubert** [comment]
Yeah, I'll try to get that done this afternoon. I had set up a trigger to build when another swagger image builds, but I guess that trigger isn't working.

**Comment by jimschubert** [comment]
I've updated these versions on Docker hub. Looks like the automation I did for new versions was local on my old machine, and I didn't migrate whatever scripts when I recently upgraded. Rather than have the updates all on my side, I've created a `versions` branch where a version number can just be bumped and tagged.

I've pushed tags for 2.3.0 and 2.3.1. Docker Hub built 2.3.1, but 2.3.0 has been queued for over 15 minutes. I'm going to go ahead and close this, since 2.3.1 is preferable anyway.

Thanks for bringing it to my attention that the tags weren't built. I probably never would have noticed it, since I made it so I that I _would_ forget about it.
